pre_commit/git.py

```diff
@@ -128,7 +128,7 @@ def get_staged_files(cwd: Optional[str] = None) -> List[str]:
         cmd_output(
             'git', 'diff', '--staged', '--name-only', '--no-ext-diff', '-z',
             # Everything except for D
```

Comment:
you'll want to update and/or remove the comment

Reply:
> you'll want to update and/or remove the comment

Removed f7cad33